src/learning_service/read_data.py

```diff
@@ -80,54 +82,56 @@ def display_data_schema(filename: str):
     full_file_path = os.path.join(DATASET_DIR, filename)
     stats_options = tfdv.StatsOptions(enable_semantic_domain_stats=True)
     stats = tfdv.generate_statistics_from_csv(
-        full_file_path,
-        delimiter='\t',
-        stats_options=stats_options
+        full_file_path, delimiter="\t", stats_options=stats_options
     )
     schema = tfdv.infer_schema(stats)
-    print("\n\n",f"\033[{MAIN_COLOR}m------------ Schema: {filename} --------------\033[0m\n")
+    print(
+        "\n\n",
+        f"\033[{MAIN_COLOR}m------------ Schema: {filename} --------------\033[0m\n",
+    )
```

Comment:
Please don't format the code unless it's required by the linting software, otherwise it will create tons of merge conflicts.